src/main/cpp/t08_cycleswap.cpp

```diff
@@ -19,5 +19,19 @@
 using namespace std;
 
 int t08_cycleswap() {
-
+    int n,x;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'x' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4493301)